test/integration/test_groups.py

```diff
@@ -0,0 +1,122 @@
+from typing import Optional
+
+from galaxy_test.base.populators import DatasetPopulator
+from galaxy_test.driver import integration_util
+
+
+class TestGroupsApi(integration_util.IntegrationTestCase):
```

Comment:
There seem to be existing API tests for groups in `lib/galaxy_test/api/test_groups.py` and there is no configuration change in Galaxy needed, so these integration tests are unnecessary.